File: frost-scripts/performance-test/nested-data-workload.py
#!/usr/bin/env python3

n = 200_000

data = [{"a": {"b": {"c": {"d": {"e": i}}}}} for i in range(n)]

result = sum(entry["a"]["b"]["c"]["d"]["e"] for entry in data)
assert result == 19_999_900_000
print(result)

File: frost-scripts/performance-test/nested-data-workload.rb
#!/usr/bin/env ruby

n = 200_000

data = (0...n).map { |i| {"a" => {"b" => {"c" => {"d" => {"e" => i}}}}} }

result = data.sum { |entry| entry["a"]["b"]["c"]["d"]["e"] }
raise unless result == 19_999_900_000
puts result
